apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -130,19 +130,14 @@
             margin-top: f.spacing(d);
         }
 
-        > .c-contentPage-img + h3,
-        > .c-contentPage-img + h4 {
-            margin-top: f.spacing(f);
-        }
-
         > p + h2 {
             margin-top: f.spacing(h);
         }
 
-        ul + h3 {
-            margin-top: f.spacing(f);
-        }
-
+        ul + h3,
```

Comment:
Not sure if there's a better way of doing this (as I'm guessing it's due to us using markdown), but selectors like this do tend to scare me a little bit as their scope is a bit vague.

Reply:
I do see the potential for scary scope, but these aren't too bad IMO, it's just for `h3`s and `h4`s that directly follow some other element types (as I'm sure you can tell 😄)

Reply:
it is scary, I'd probably try to limit it by only applying to immediate children using `>`. It _could_ be that there is a way to prepend elements created by the markdown renderer with a class which would solve this

Reply:
I've not added this particular rule, only moved it, but I'll see if it all still behaves with the added `>`